src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -165,12 +165,13 @@ def __init__(
         body: dace.SDFG,
         state: dace.SDFGState,
         symbol_map: dict[str, IteratorExpr | ValueExpr | SymbolExpr],
+        **kwargs,
```

Comment:
I'd prefer defaulted parameters instead of `**kwargs`:
```suggestion
        reduce_limit = 0,
        reduce_wcr = None,
```